02.general/02.how-to-create-your-own-ad-filters/docs.en.md

```diff
@@ -77,6 +77,7 @@ visible: true
         * [Pseudo-class `:properties()`](#extended-css-properties)
         * [Selectors debugging mode](#selectors-debugging-mode)
         * [Testing extended selectors](#testing-extended-selectors)
+        * [Pseudo-property `remove`](#pseudo-property-remove)
```

Comment:
The order is wrong -- you have it before "testing extended selectors" in the article text

Reply:
changed the order